decentraland/unity-renderer
# Fix: Remove the nonsense button that appears in the nav map

## What does this PR change?

Remove the nonsense button that appears in the nav map



## How to test the changes?

Open the map, and check if the button is not there.
![image](https://user-images.githubusercontent.com/10808443/202248890-7a41f721-bc69-4c64-80a7-47f0d6f6bb85.png)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
